controllers/Router.php

```diff
@@ -56,4 +56,26 @@ public function cleanUri($url): string {
 		return UriNormalizer::normalize($uri,UriNormalizer::REMOVE_DUPLICATE_SLASHES);
 	}
 
+	public function getMethod(array $method, $controller) {
+		if (empty($controller)){
+			$controller = 'Home';
+			$method = "home";
+		}
+		$controllerClass = "OC_Blog\\Controllers\\Controller" . $controller;
+
+		if (method_exists("$controllerClass", $method[2]) ) {
+			return $method[2];
+		}else{
```

Comment:
Codacy found an issue: [The method getMethod uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)